engine/access/rpc/backend/backend_block_details.go

```diff
@@ -57,7 +54,7 @@ func (b *backendBlockDetails) GetLatestBlock(ctx context.Context, isSealed bool)
 		return nil, flow.BlockStatusUnknown, status.Errorf(codes.Internal, "could not get latest block: %v", err)
 	}
 
-	stat, err := b.getBlockStatus(ctx, block)
+	stat, err := b.getBlockStatus(ctx, block.Header)
```

Comment:
let's update these to use the full word for clarity
```suggestion
	status, err := b.getBlockStatus(ctx, block.Header)
```